pom.xml

```diff
@@ -116,7 +116,7 @@
     <htrace.version>3.1.0-incubating</htrace.version>
     <collections.version>3.2.2</collections.version>
     <!-- Do not change jodatime.version until HBASE-15199 is fixed -->
-    <jodatime.version>1.6</jodatime.version>
+    <jodatime.version>2.0</jodatime.version>
```

Comment:
HBASE-15199 has fix version 2.0.0 so this may screw up something for 4.x. Why do we need this?

Reply:
I was hitting this exception : https://stackoverflow.com/questions/36358290/aws-issue-with-joda-time-java-lang-nosuchmethoderror without latest version of joda. I thought it should be okay for demo at least like @dbwong mentioned.

Reply:
Sounds good, as long as our select query/upsert etc. doesn't fail for the demo